subprojects/model-core/src/main/java/org/gradle/internal/reflect/validation/DefaultTypeAwareProblemBuilder.java

```diff
@@ -41,20 +41,20 @@ public DefaultTypeAwareProblemBuilder(BuildOperationProgressEventEmitter buildOp
     @Override
     public TypeAwareProblemBuilder withAnnotationType(@Nullable Class<?> classWithAnnotationAttached) { // TODO (donat) figure out how all functions can return TypeAwareProblemBuilder
```

Comment:
Unrelated to this PR, but does this have an issue?
This will get lost in no time.

Reply:
I think that that comment is no longer relevant.